hw/ip/pattgen/data/pattgen.hjson

```diff
@@ -0,0 +1,148 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "pattgen"
+  clock_primary: "clk_fixed"
+  bus_device: "tlul"
+  bus_host: "none"
+  estimated_gate: "TBD"
+  // OUTPUT pins
+  available_output_list: [
+    { name: "pda1_tx", desc: "Serial output data bit for pattern generation on channel 1" }
+    { name: "pcl1_tx", desc: "Bit-banging output clock for pattern generation on channel 1" }
+    { name: "pda2_tx", desc: "Serial output data bit for pattern generation on channel 2" }
+    { name: "pcl2_tx", desc: "Bit-banging output clock for pattern generation on channel 2" }
+  ]
+  // INTERRUPT pins
+  interrupt_list: [
+    { name: "pattern_done"
```

Comment:
I know the one pager says one interrupt, but I'm thinking
there should probably be one per group?

Reply:
It makes sense, I have to update the description INTR_MASK reg